worker/tests/test_upsert_metadata.py

```
@@ -0,0 +1,29 @@
+
+import pytest
+
+from src.db import init_db, get_connection
+from src.config import get_settings
+from src.jobs.upsert_document import upsert_document
+from .test_upsert_document import FakeRelay
+
+
+@pytest.mark.asyncio
+async def test_metadata_persisted_on_create(tmp_path):
+    db = str(tmp_path / "t.db"); init_db(db); conn = get_connection(db)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)